src/system/Form/Toggle.stories.jsx

```diff
@@ -91,6 +92,20 @@ const CustomStyling = args => (
 	</form>
 );
 
+const ToggleTextStyling = args => (
+	<form>
+		<ToggleText
+			toggleOnLabel="Table"
+			toggleOffLabel="Chart"
+			checked={ args.checked }
+			disabled={ false }
+			variant="info"
+		/>{ ' ' }
```

Comment:
If our goal is to add something that selects `A` or `B`, we probably would be better off with a component like this:

```jsx
<ToggleText
  states=["A", "B"]
  value="A"
  onChange={...}
/>
```

This could later be extended to add more states, if need arises. Just a thought.

Reply:
> But if the choice is always going to be binary, is there any advantage of providing an array like states ?

My two cents:

I feel like the switch component is good for setting something as enabled / disabled: `View as table: <switch here>`
But the text switch implies we're choosing from a list of options (albeit only two): `View as: "chart" | "table"` which I guess is where the states come in?

Maybe if we only want to support a binary choice we could save ourselves some effort and just use the existing switch component as is with an appropriate label to indicate one of the options is enabled or disabled: `View as table: <switch here>`?

Alternatively I had a quick look and I think styling the [RadioButtonGroup](https://vip-design-system-components.netlify.app/?path=/docs/radioboxgroup--docs) component according to the screenshot would be a light lift and makes more sense to me personally for this use-case.

Or we could add a new component to the design system based on this: https://www.radix-ui.com/primitives/docs/components/toggle-group? 

Curious what the Patisserie folks think.

Reply:
Yeah, I was wondering the same. The same component we're using for time ([the inline-select](https://vip-design-system-components.netlify.app/?path=/story/form-select--is-inline&args=label:View+as;options[0].label:Table;options[0].options:!undefined;options[0].value:table;options[1].label:Chart;options[1].options:!undefined;options[1].value:Chart)) could work here and is a much more habitual pattern than the toggle between two options. 

We would get all the accessibility done for us and the pattern is common in the dashboard (and elsewhere), so there would be no risk of confusing the user.

[The linked example will show how it'll look with the two options.](https://vip-design-system-components.netlify.app/?path=/story/form-select--is-inline&args=label:View+as;options[0].label:Table;options[0].options:!undefined;options[0].value:table;options[1].label:Chart;options[1].options:!undefined;options[1].value:Chart)
![CleanShot 2023-10-05 at 15 39 19@2x](https://github.com/Automattic/vip-design-system/assets/668251/8c5467f6-6873-4d40-8736-ff8b87460586)

Reply:
Thanks folks! I'm going to timebox and give the toggle group and radio button group elements a go. 

@andrea-sdl : we're already currently using the inline select but I guess I wanted to take a stab at implementing the new toggle design prescribed in the Figma

<img width="582" alt="image" src="https://github.com/Automattic/vip-design-system/assets/14042477/f174db43-4335-4dad-b902-e23414bc15d3">

Reply:
Wonder if we should just stick with this.